io-ballerina/print.bal

```diff
@@ -22,6 +22,11 @@ import ballerina/jballerina.java;
 # 3. `io:PrintableRawTemplate` - an raw templated value
 public type Printable any|error|PrintableRawTemplate;
 
+# Holds the output streaming types:
```

Comment:
```suggestion
# Defines the output streaming types.
```

Reply:
make it consistent with Printable

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/222/commits/7f80302bc485e03a13abb7d4fdaa183fe48a1674